<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogArea</class>
 <widget class="QDialog" name="dialogArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gra</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>671</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/dialog background.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureOfCharacter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Backgorund-start-menu.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameOfCharacterLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>24</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#cfcfcf;&quot;&gt;NameOfCharacter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialogCharacterLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>571</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>24</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#cfcfcf;&quot;&gt;Dialog&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureOfEnemy">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>180</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Backgorund-start-menu.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameOfAI">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>24</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#cfcfcf;&quot;&gt;NameOfAI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialogAILabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>24</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#cfcfcf;&quot;&gt;DialogAI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose1Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>478</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Choose 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose3Button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>478</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Choose 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose2Button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>478</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Choose 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
